packages/metrics/src/model/mutant-model.ts

```diff
@@ -12,6 +12,7 @@ function assertSourceFileDefined(sourceFile: FileUnderTestModel | undefined): as
  * Represent a model of a mutant that contains its test relationship
  */
 export class MutantModel implements MutantResult {
+  [key: string]: any;
```

Comment:
This was necessary to fix compile errors, but is there a better way?

https://github.com/stryker-mutator/mutation-testing-elements/pull/2453/files#diff-0f3462bf2afd2ec63a1c6a56844b694f192bcba2798139a072110019706b4337R223-R225